8.14/build.gradle

```diff
@@ -0,0 +1,6 @@
+buildPlugin(this, '8.14', revisions(0))
```

Comment:
Since the latest version is `8.14.3` you should use `revisions(3)` here.

Reply:
You can also look at earlier versions, at least `8.13` to see if anything should be bumped.